sanic/pages/error.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Mapping
+
+import tracerite.html  # type: ignore
```

Comment:
We should not be ignoring this. If there is an import issue we should handle it.